modules/logging-release-notes-5-8-12.adoc

```diff
@@ -0,0 +1,63 @@
+// module included in logging-5-8-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="cluster-logging-release-notes-5-8-12_{context}"]
+= Logging 5.8.12
+This release includes a link:https://access.redhat.com/errata/RHBA-2024:6459[OpenShift Logging Bug Fix Release 5.8.12] and link:https://access.redhat.com/errata/RHBA-2024:6094[OpenShift Logging Bug Fix Release 5.8.12].
+
+[id="openshift-logging-5-8-12-bug-fixes_{context}"]
+== Bug fixes
+
+* Before this update, the collector used internal buffering with a `drop_newest` setting to reduce high memory usage, which led to significant log loss. With this update, the collector returns to its default behavior to prevent this issue. (link:https://issues.redhat.com/browse/LOG-6026[LOG-6026])
```

Comment:
What was the default behavior and why did it change to internal buffering?

Reply:
default behavior: buffer is customized. it is what is designed for the output plugin in upstream.